### ISSUE TITLE
Not able to apply --ui-highlight-preset

### ISSUE THREAD

**Issue by robguinness** [mthenw/frontail#149]
I'm not sure if I'm missing something obvious, but I'm not able to apply the --ui-highlight-preset option. I'm assuming that the path can be given relative to the current working directory, but I also tried using the full path to the json file, and it still had no effect. Can you please provide an example or something???

**Comment by mthenw** [comment]
Hey, thanks for reporting. You are right. It doesn't work. Let me check it.

**Comment by mthenw** [comment]
Yeah, I probably know what's the issue. If you use `--ui-highlight-preset` flag you also need to use `--ui-highlight`. That's probably not the greatest design so I'm gonna mark it as a thing to fix in v5.

**Comment by robguinness** [comment]
Thanks, I can confirm that it is working now as expected with this change. Looking forward to v5!
